### DISCUSSION TITLE
How can I increase priorty for h{2,6}

### DISCUSSION THREAD

**Discussion by tenmozes** [Pagefind/pagefind#246] [General]
Context:
- Hugo as static site engine 
- All content in markdown (basically documentation site)
- Many h2-h6 per page

There are a lot of entries with generic words like `export`. When I try to do a lookup by `export` word, I get all results from pages however, there is a dedicated section about exporting. How to prioritize it?

I've read the documentation (metadata section). It looks like the solution there, but I didn't find it :) 

Thank you

**Comment by bglw** [comment]
Hi @tenmozes 👋 

There isn't yet a way to achieve this, but it's coming soon! #13 is the issue to follow for this, and it's one of the next things on my plate 🙂
